modules/nw-router-configuring-dual-stack.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * networking/routes/route-configuration.adoc
+
+[id="nw-router-configuring-dual-stack_{context}"]
+= Configuring the {product-title} router for dual-stack support
+
+When you install a {product-title} cluster on bare metal infrastructure, you can configure the router for single-stack or dual-stack support.
+
+.Prerequisites
+
+* You deployed an {product-title} cluster on bare metal.
+* You installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+////
+. Open the Network YAML file by entering the following command:
++
+[source,terminal]
+----
+$ oc get network/cluster -o yaml
+----
+////
+
+. To configure the router for single-stack or dual-stack support, create a service:
++
+[source,terminal]
+----
+$ oc get service <project_name>
+----
```

Comment:
Need correct command. Does this require a new service or is it better to modify an existing service?

Should this be clarified as a `clusterIP` service?